src/Shared/UnitTests/TypeLoader_Dependencies_Tests.cs

```diff
@@ -0,0 +1,138 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+
+using System;
+using System.IO;
+using Microsoft.Build.Shared;
+using Microsoft.Build.SharedUtilities;
+using Xunit;
+
+namespace Microsoft.Build.UnitTests
+{
+    public class TypeLoader_Dependencies_Tests
```

Comment:
When you put it like that, it makes sense :)

I guess one point of improvement would be to make some test utilities around task loading to reduce code duplication. But I'm fine without it as well.